packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { cloneElement, forwardRef, ReactElement, Ref } from 'react';
+import React, { cloneElement, forwardRef, MouseEventHandler, ReactElement, Ref } from 'react';
```

Comment:
Is it really in use?

Reply:
Doh, i add it as some point to type a bit the events in my iteration but there is no use for it now.
I remove it